content/applications/general/integrations/translate.rst

```diff
@@ -0,0 +1,122 @@
+
+Click on :guilabel:`Create` to finish this step.
+
+API Library
+-----------
+
+Next, the :guilabel:`Cloud Translation API` needs to be installed on this newly-created project.
+To do that, click :guilabel:`Library` in the left menu. Then, search the term `Cloud Translation
+API`, and click into the result. This should be a *Google Enterprise API* labeled `Cloud Translation
+API`.
+
+Click :guilabel:`Enable` to install the library on this project.
+
+.. important::
+   Using the *Google Translate* API **requires** a current billing account with `Google
+   <https://https://myaccount.google.com/>`_.
+
+Once a billing account is setup with *Google* and the library is enabled, click :guilabel:`Manage`
+to finish configuration on the API.
+
+Create Credentials
+------------------
+
+Now that the project is set up, and the library is enabled, credentials **must** be created. This
+includes the *API key*.
+
+To begin this process, click :guilabel:`Credentials` in the left sidebar menu.
+
+Then, click :guilabel:`Create Credentials` in the top menu, and select :guilabel:`API key` from the
+drop-down menu.
+
+.. image:: translate/api-key.png
+   :align: center
+   :alt: Create an API key in the Google developer console.
+
+Copy the :guilabel:`API key` for use in the next section.
+
+.. warning::
+   For security purposes, the usage of the *API key* can be restricted.
+
+   To do that, go to the :guilabel:`API restrictions` by clicking on :guilabel:`Edit API key` in the
+   pop-over window, or on the listed :guilabel:`API key` on the :guilabel:`Credentials` page. From
+   here, key restrictions can be set. This includes setting an application to restrict the use of
+   the API key, and whether this *API key* can call any API.
+
+   It is recommended that the Odoo *Translate API* be restricted to **only** allow requests from the
+   configured database and to the `Cloud Translation API`.
+
+   To add the website restriction, click :guilabel:`Websites`, under the :guilabel:`Set an
+   application restriction`. Then, enter the address of the database *Google Translate* is being
+   used in, by clicking on :guilabel:`Add`. Lastly, add the :abbr:`URL (Uniform Resource Locator)`,
+   and click :guilabel:`Done`.
+
+   To restrict use of the key to a selected API, first, select :guilabel:`Restrict key`, under the
+   :guilabel:`API restrictions` section. Then use the drop-down menu to choose the API being
+   configured (`Cloud Translation API`).
+
+.. tip::
+   - Save the API Key: copy the API key and securely store it.
+   - Do **not** share it publicly or expose it in client-side code.
+
+Odoo set up
+===========
+
+To access the integration in Odoo, navigate to the :menuselection:`Settings app --> Discuss section
+--> Message Translation`. Enter the :guilabel:`API key` into the field labeled :guilabel:`Paste your
+API key`. Then, :guilabel:`Save` the settings, and *Google Translate* can be used in any chatter
+throughout the database.
```

Comment:
The field name is **Message Translation** and the palceholder text within the field is "Paste your API key".
```suggestion
To access the integration in Odoo, navigate to the :menuselection:`Settings app --> Discuss
section`. Enter the API key into the field labeled :guilabel:`Message Translation`. Then,
:guilabel:`Save` the settings, and *Google Translate* can be used in any chatter throughout the
database.
```